### DISCUSSION TITLE
스프라이트 이미지 관련 툴 정보 공유

### DISCUSSION THREAD

**Discussion by sryung1225** [marketkarly-team/marketkarly-client#24] [General]
아까 잠깐 얘기드린 sprite 생성 패키지에요!
> https://www.npmjs.com/package/gulp.spritesmith

제가 현업시절에 사용하던 방식인데 (gulp 태스크 러너 이용)
이미지들을 한 폴더에 모아두면 그걸 하나의 sprite 이미지로 만들고, 이미지의 각각의 좌표를 저장한 scss(mixin) 파일을 함께 만들어줍니다. scss를 사용할 기회가 생겼을 때 한 번 떠오를 수 있기를! :)

<br>

개인이 쉽게 만드는 데에는 아래 사이트 추천합니다!
> 이미지 생성 용 : https://www.toptal.com/developers/css/sprite-generator/
> 이미지 좌표 확인 용 : http://www.spritecow.com/

**Comment by jaesukpark77** [comment]
정말 유용한 정보인 것 같습니다.

저도 꼭 scss 활용하는 경우에 적용해 보겠습니다!!!👍👍👍
